pallas-network/tests/protocols.rs

```diff
@@ -399,6 +401,150 @@ pub async fn local_state_query_server_and_client_happy_path() {
     _ = tokio::join!(client, server);
 }
 
+#[tokio::test]
+pub async fn local_state_query_server_and_block_query_get_epoch_client_happy_path() {
```

Comment:
there's no need to repeat the whole code of the test for each type of query. Put everything in a single happy path test that includes all of the available queries.

Reply:
@scarmuega done.